lib/wallaby/query.ex

```diff
@@ -235,13 +236,17 @@ defmodule Wallaby.Query do
     Keyword.get(conditions, :count)
   end
 
+  def getNumber(%Query{conditions: conditions}) do
+    Keyword.get(conditions, :get)
+  end
+
   def inner_text(%Query{conditions: conditions}) do
     Keyword.get(conditions, :text)
   end
 
   def result(query) do
     cond do
-      count(query) == 1 ->
+      count(query) == 1 || getNumber(query) != nil ->
```

Comment:
Agreed. Done.
Actually I moved the whole condition into the function because asking for an element which has only one instance is just another way of asking for a specific element, and this makes the `cond` easier to read.
Not sure of the best location for the function but in the end put it immediately after its use.